src/Controllers/Quotes/randomQuote.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Allows the viewing of a single random quote
+ */
+use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\ServiceAPI\MyRadio_Quote;
+
+CoreUtils::getTemplateObject()->setTemplate('table.twig')
+    ->addVariable('title', 'Random Quote')
+    ->addVariable('tabledata', CoreUtils::setToDataSource(MyRadio_Quote::getRandom()))
+    ->addVariable('tablescript', 'myradio.quotes')
```

Comment:
You don't seem to have created the myradio.quotes.js this is referring to :)

Reply:
It uses the same tablescript as the main quotes board page

Reply:
D'oh!